x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/user/RestGetUserPrivilegesAction.java

```diff
@@ -81,8 +81,8 @@ public RestResponse buildResponse(GetUserPrivilegesResponse response, XContentBu
 
             builder.field(RoleDescriptor.Fields.CLUSTER.getPreferredName(), response.getClusterPrivileges());
             builder.startArray(RoleDescriptor.Fields.GLOBAL.getPreferredName());
-            for (ConditionalClusterPrivilege ccp : response.getConditionalClusterPrivileges()) {
-                ConditionalClusterPrivileges.toXContent(builder, ToXContent.EMPTY_PARAMS, Collections.singleton(ccp));
+            for (ConfigurableClusterPrivilege ccp : response.getConditionalClusterPrivileges()) {
+                ConfigurableClusterPrivileges.toXContent(builder, ToXContent.EMPTY_PARAMS, Collections.singleton(ccp));
```

Comment:
`ConfigurableClusterPrivileges.toXContent(builder, ToXContent.EMPTY_PARAMS, response.getConditionalClusterPrivileges());`
This is important!

Reply:
You're right, I saw something looking broken, and I assumed it's introduced in this PR. My bad, sorry.
Thank you, Yogesh!